wiki/Tournaments/MWC/2019_7K/en.md

```diff
@@ -86,10 +86,57 @@ The osu!mania 7K World Cup 2019 is run by various community members by distribut
 
 ------------------------------------------------------------------------
 
+## Match Schedule: Group Stage
+
+### Saturday, 9. February 2018
```

Comment:
Minor issue but the teams should be higher seed on the right and lower seed on the left side.